packages/react-test-renderer/src/ReactTestRenderer.js

```diff
@@ -366,6 +369,9 @@ function toTree(node: ?Fiber) {
     case HostText:
       return node.stateNode.text;
     case Fragment:
+    case ContextProvider:
+    case ContextConsumer:
+    case Mode:
```

Comment:
I'm treating these as fragments right now because I'm not really sure how this output is consumed. Should I create a new `nodeType`?

Reply:
Seems like `nodeType` is only used for user-defined components and host components. Everything else is treated as a pass-through.